### PR TITLE
feat: Initial generation of google-iam-credentials-v1

### PR DESCRIPTION
**Pull request by dazuma**
Auto-created at 2020-12-09 19:52:06 +0000 using `toys gen-client`.

### CODE REVIEW

File: google-iam-credentials-v1/synth.py [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+    extra_proto_files=[
+        "google/cloud/common_resources.proto",
+    ],
+    generator_args={
+        "ruby-cloud-gem-name": "google-iam-credentials-v1",
+        "ruby-cloud-title": "IAM Service Account Credentials V1",
+        "ruby-cloud-description": "The Service Account Credentials API creates short-lived credentials for Identity and Access Management (IAM) service accounts. You can also use this API to sign JSON Web Tokens (JWTs), as well as blobs of binary data that contain other types of tokens.",
+        "ruby-cloud-env-prefix": "IAM_CREDENTIALS",
+        "ruby-cloud-grpc-service-config": "google/iam/credentials/v1/iamcredentials_grpc_service_config.json",
+        "ruby-cloud-product-url": "https://cloud.google.com/iam",
```
Comment (quartzmo) [inline google-iam-credentials-v1/synth.py:37]: I looked for a better product page and found this: https://cloud.google.com/iam/docs/reference/credentials/rest

Unfortunately it's for the REST API.

**Review by quartzmo — APPROVED** [review]
`IAM_CREDENTIALS_CREDENTIALS` lol

**Comment by dazuma** [comment]
> `IAM_CREDENTIALS_CREDENTIALS` lol

Yeah... 🙈

File: google-iam-credentials-v1/synth.py (hunk 1 above)
Comment (dazuma) [reply google-iam-credentials-v1/synth.py:37]: I did look at that, and decided against it because yes, it's for the REST API.

**Pull request merged**